package.json

```diff
@@ -10,16 +10,18 @@
   "author": "Thomas Jenkinson <tjcode@outlook.com>",
   "license": "MIT",
   "repository": "tjenkinson/clappr-thumbnails-plugin",
+  "dependencies": {
+    "clappr": "latest"
```

Comment:
I tried everything to make it work. But it really is not a dependency. I forgot to remove